apps/dashboard/src/@/components/ui/form.tsx

```diff
@@ -93,7 +93,7 @@ const FormLabel = React.forwardRef<
   return (
     <Label
       ref={ref}
-      className={cn(error && "text-destructive-foreground", className)}
+      className={cn(error && "text-destructive", className)}
```

Comment:
why this change?

Reply:
`destructive-foreground` is just white / whiteish vs destructive is red